src/Aspire.Hosting.Nats/NatsBuilderExtensions.cs

```diff
@@ -30,19 +31,28 @@ public static IResourceBuilder<NatsServerResource> AddNats(this IDistributedAppl
     /// <summary>
     /// Adds JetStream support to the NATS server resource.
     /// </summary>
-    /// <param name="builder">NATS resource builder.</param>
-    /// <param name="srcMountPath">Optional mount path providing persistence between restarts.</param>
-    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<NatsServerResource> WithJetStream(this IResourceBuilder<NatsServerResource> builder, string? srcMountPath = null)
```

Comment:
What's the policy on making API breaking changes right now? Is this supposed to be left in and `[Obsolete]`?

Reply:
good point. marked as obsolete. I would've thought api changes are tolerable when in preview so I'd be happy to remove if you're OK with it.